python/pytests/aggregation/max_test.py

```diff
@@ -0,0 +1,33 @@
+import kaskada as kd
+
+
+def test_max_unwindowed(source, golden) -> None:
```

Comment:
How are these executing? It doesn't look like they have a source defined?

Reply:
Yup, I addressed these comments. Please take another look